frontend/javascripts/oxalis/model/sagas/isosurface_saga.js

```diff
@@ -387,11 +387,11 @@ function* downloadIsosurfaceCellById(cellId: number): Saga<void> {
   stl.setUint32(cellIdIndex, cellId, true);
 
   const blob = new Blob([stl]);
-  yield* call(saveAs, blob, `mesh-${cellId}.stl`);
+  yield* call(saveAs, blob, `${cellName}-${cellId}.stl`);
 }
 
 function* downloadIsosurfaceCell(action: TriggerIsosurfaceDownloadAction): Saga<void> {
-  yield* call(downloadIsosurfaceCellById, action.cellId);
+  yield* call(downloadIsosurfaceCellById, action.cellName, action.cellId);
```

Comment:
The CI still fails because `yarn flow` is not happy :)

In line 399 there is another call to `downloadIsosurfaceCellById` which still uses the old signature. However, good news: `downloadActiveIsosurfaceCell` seems to be dead code. Originally, this function was used in the segments tab:

https://github.com/scalableminds/webknossos/blob/ed0c278efb1fdd64d4bcd0b1427243af87b48b47/frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.js#L161-L161

However, it's not used anymore. Hence, you can safely remove:
- downloadIsosurface
- TriggerActiveIsosurfaceDownloadAction
- triggerActiveIsosurfaceDownloadAction
- downloadActiveIsosurfaceCell
- and related lines, such as imports and `_takeEvery` calls
I'll doublecheck again that not too much was removed :)